packages/path_provider/path_provider_linux/pubspec.yaml

```diff
@@ -11,17 +11,17 @@ flutter:
         pluginClass: none
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
   flutter: ">=1.10.0 <2.0.0"
 
 dependencies:
-  path: ^1.6.4
-  xdg_directories: ^0.1.0
-  path_provider_platform_interface: ^1.0.1
+  path: ^1.8.0-nullsafety.3
+  xdg_directories: ^0.1.2
```

Comment:
The PR to migrate xdg_directories to nnbd is here https://github.com/flutter/packages/pull/250